src/pages/Aggregator/AggregatorInfo/AggregatorInfo.types.ts

```diff
@@ -7,8 +7,11 @@ export type AggregatorInfoContainerProps = {
 
 export type AggregatorInfoComponentProps = {
   onClick: () => void;
-  state: FiniteStates;
+  feesAndLimitsState: FiniteStates;
   feesAndLimits: FeesAndLimitsType;
   tokenName?: string;
   tokenDecimals?: number;
+  incentivesState?: FiniteStates;
+  incentives?: IncentivesType;
```

Comment:
Why are these optional? You always have `incentivesState` (default value is `idle`) and `IncentivesType` is the same as `FeesAndLimitsType` (both contains optional properties).